drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -227,12 +227,17 @@ function DragAndDropTemplates(configuration) {
     };
 
     var feedbackTemplate = function(ctx) {
-        var feedback_display = ctx.feedback_html ? 'block' : 'none';
         var properties = { attributes: { 'aria-live': 'polite' } };
+        var messages = ctx.overall_feedback_messages || [];
+        var feedback_display = messages.length > 0 ? 'block' : 'none';
+        var feedback_messages = messages.map(function(message) {
+            return h("p.message", {innerHTML: message}, []);
```

Comment:
@e-kolpakov Nit: I don't think this needs to pass `[]` to `h` to indicate that the element shouldn't have any children. Not passing a third argument has the same effect.